vizro-core/src/vizro/models/_components/figure.py

```diff
@@ -0,0 +1,66 @@
+from typing import Literal
+
+from dash import dcc, html
+
+try:
+    from pydantic.v1 import Field, PrivateAttr, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, PrivateAttr, validator
+
+import vizro.figures as vf
+from vizro.managers import data_manager
+from vizro.models import VizroBaseModel
+from vizro.models._components._components_utils import _callable_mode_validator_factory, _process_callable_data_frame
+from vizro.models._models_utils import _log_call
+from vizro.models.types import CapturedCallable
+
+
+# TODO: Align on naming - really difficult one:
+# ReactiveComponent, Reactive, Figure, ReactiveFigure, ReactiveObject, ReactiveHTML, ReactiveDiv, any other ideas?
+class Figure(VizroBaseModel):
```

Comment:
I personally find `Controllable` even more confusing 😄 In the end, what do we mean with Reactive? It's reactive to **controls and actions** in a sense that it can be the output of any of these, so it's not only controls. I also don't like `Reactive` as a prefix because I think the prefix is redundant. All our components are reactive except for the `Card,` we do not call them ReactiveGraph, ReactiveTable, etc., because that's what you would expect anyway.

Going back to Antony's comment here: https://github.com/mckinsey/vizro/pull/493#pullrequestreview-2100342271, I also don't think there will be a need for a higher-level model called `UnreactiveComponent `or something like this, which is why I prefer to not have the prefix.

Reply:
Let's go with `Figure` then, it's probably the best choice ultimately 👍